cpp/examples/bfs_father_example.cc

```diff
@@ -36,8 +36,8 @@ int main(int argc, char* argv[]) {
   auto maybe_vertices =
       GAR_NAMESPACE::ConstructVerticesCollection(graph_info, label);
   ASSERT(maybe_vertices.status().ok());
-  auto& vertices = maybe_vertices.value();
-  int num_vertices = vertices.size();
+  auto vertices = maybe_vertices.value();
```

Comment:
Maybe use `auto&` consistently is better?

Reply:
Fixed.